### DISCUSSION TITLE
Cannot post logs to the server (file is being used by another program)

### DISCUSSION THREAD

**Discussion by Cyb3r-Monk** [cmu-sei/GHOSTS#111] [Q&A] [answered]
In app.log, I keep seeing the below error:
Problem posting logs to server: The process cannot access the file 'C:\ghosts\logs\clientupdates.log' because it is being used by another process.

Is there a way to solve this issue?

**Comment by sei-dupdyke** [comment]
This is just because the last cycle has not completed yet, I'll fix this in next update by adding a "wait until the file is unlocked" loop.

**Reply by Cyb3r-Monk** [reply]
What causes the cycle not complete? If I understand correctly, the below config causes the cycle run forever. Is that true?
"UtcTimeOn": "00:00:00",
"UtcTimeOff": "24.00:00:00",
"HandlerArgs": {},
"Loop": true

**Comment by sei-dupdyke** [comment, marked as answer]
Timeline handers are all on their own threads. Same for sending logs home. So, if sending logs back home gets backed up, this is the cycle I'm referring to, it has nothing to do with other activities GHOSTS is coordinating. Sorry for the confusion.

**Reply by Cyb3r-Monk** [reply]
Oh, I see. I'll wait for the next update then. Thanks a lot!
